src/map/attackround.cpp

```diff
@@ -51,27 +51,30 @@ CAttackRound::CAttackRound(CBattleEntity* attacker, CBattleEntity* defender)
         m_coverAbilityUserEntity = nullptr;
     }
 
-    // Build main weapon attacks.
-    CreateAttacks(dynamic_cast<CItemWeapon*>(attacker->m_Weapons[SLOT_MAIN]), RIGHTATTACK);
-
-    // Build dual wield off hand weapon attacks.
-    if (IsH2H())
+    if (IsH2H()) // Build H2H attacks.
     {
-        // Build left hand H2H attacks.
         CreateAttacks(dynamic_cast<CItemWeapon*>(attacker->m_Weapons[SLOT_MAIN]), LEFTATTACK);
-
-        // Build kick attacks.
-        CreateKickAttacks();
+        CreateAttacks(dynamic_cast<CItemWeapon*>(attacker->m_Weapons[SLOT_MAIN]), LEFTATTACK);
```

Comment:
To passers-by: this was gross before these changes were made and `CreateAttacks` has null-checks built in. Do not learn from this code though, this isn't how to use `dynamic_cast`!

(you don't need to change anything... unless you're feeling brave)

Reply:
Arent we building 2 left-hand attacks here though?
I would think H2H attacks would build one right and one left.
That was what was happening before the changes.

Reply:
> To passers-by: this was gross before these changes were made and `CreateAttacks` has null-checks built in. Do not learn from this code though, this isn't how to use `dynamic_cast`!
> 
> (you don't need to change anything... unless you're feeling brave)

I was feeling brave this morning, hehe. Sorry about that :D